eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -81,6 +81,25 @@ stages:
       overrideDistroDisablingSha1: false # 🚫
       runOnline: true                    # ✅
       withPreviousSDK: false             # 🚫
+      useMonoRuntime: false              # 🚫
+
+  - template: ../jobs/vmr-build.yml
```

Comment:
Now that we have a passing run in PR validation, can you switch this back to only run in non-PR scenarios?  TIA